docs/use/affinity.md

```diff
@@ -0,0 +1,170 @@
+
+### Node affinity
+
+Node affinity determines which nodes a pod can be scheduled on. Based on the labels assigned to nodes, you can define rules about where a pod should or should not be deployed. 
+
+Nodes in a Kubernetes cluster have labels that consist of **key-value** pairs.
+
+If you're looking to dive deep into node affinity, check out the Kubernetes [documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#node-affinity).
+
+### Pod affinity
+
+Pod affinity enables you to control the placement of pods based on the location of other pods. By using affinity rules, you can ensure that pods are scheduled together (co-located) on the same node, in the same zone, or within any other topological boundary.
+
+Kubernetes uses labels, which are key-value pairs, to identify and categorize pods.
+
+If you're looking to dive deep into pod affinity, check out the Kubernetes [documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
+
+### Pod anti-affinity
+
+Pod anti-affinity empowers you to take control of pod placement in your Kubernetes environment. With pod anti-affinity you can prevent certain pods from being scheduled on the same node or within the same topology, such as a zone or region.
+
+Like pod affinity, Pod anti-affinity also uses pod labels to identify and match pods that should not be co-located.
+
+If you're looking to dive deep into pod anti-affinity, check out the Kubernetes [documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#inter-pod-affinity-and-anti-affinity).
+
+
+
+### Set up affinity rules for your cluster
+
+You can set affinity rules in Percona Everest via the **Advanced Configurations** page within the **Affinity** section. 
+{.power-number}
+
+1. From the Percona Everest home page, navigate through the **Create Database** wizard until you reach the **Advanced Configurations** page.
+
+2. From the **Advanced Configurations** page, click **Create affinity rule**. The **Add rule** screen will be displayed.
+
+    ![!image](../images/create_affinity.png)
+
+    
+3. Enter the information on the **Add Rule** screen. Click **Add rule**. You can view the affinity rule on the components page of the corresponding database.
+
+    ![!image](../images/add_rule_affinity.png)
+
+The following table provides details about the different attributes involved in setting up affinity rules for your databases.
+
+|**Attributes**    |**Description**  |**Note**  |
+|----------------|-----------------|--------------|
+|**Components**|<ul><li>Config Server</li><li>DB Node</li><li>Router</br></br></li><li>DB Node</br><br></li><li>DB Node</li><li>Proxy</li></ul>|Applicable for **MongoDB sharded cluster**.<br/></br></br></br></br>Applicable for **MongoDB non sharded cluster**.</br></br>Applicable for **MySQL** and **PostgreSQL** databases.|
```

Comment:
I don't feel strongly opinionated about this but I think we could make the table simpler to understand if we go with something like:
* DB node
* Proxy/Router/PGBouncer - Applicable for MySQL, MongoDB sharded clusters and PostgreSQL databases.
* Config Server - Applicable for MongoDB sharded clusters.